### ISSUE TITLE
Automatically update 'To:' field in ikhal if start date is after end date

### ISSUE THREAD

**Issue by pfertyk** [pimutils/khal#528]
When I edit an event in ikhal (say, I want to move it from today to tomorrow) I can modify both the start date ('From:') and the end date ('To:'). But I have to modify both of them. If I only change the start date and hit Save, I get the message: `Can't save: end date is before start date!`.

It is obvious that the end date must be after the start date. Would it be possible for ikhal to detect that a user modified the start date and, if the end date is now before the start date, update the end date automatically (that is: set the newly inputed start date as end date)?

I don't know if this feature would be useful for other users too, but it seems that way to me. Also, I don't know if it's possible in urwid to react on events (such as 'user selected end date field' or 'user has left start date field'), so I don't know how much problem would it be to implement this.

**Comment by geier** [comment]
We had something like this once, but it didn't work properly, so we removed it for the time being. But I believe this would be a nice enhancement to have.

**Comment by pfertyk** [comment]
Why was it not working? Was the functionality itself not as useful as it seems or was there a problem with implementation?

**Comment by geier** [comment]
I don't believe that feature ever hit the master branch, see https://github.com/pimutils/khal/commit/de41f8826d19c7346f76222f39cc14a01ed8633f for a commit that implements something like this, and https://github.com/pimutils/khal/issues/335#issuecomment-184345576 for a brief discussion.
